pkg/processor/cascadingfilterprocessor/processor_test.go

```diff
@@ -401,16 +405,19 @@ func TestMultipleBatchesAreCombinedIntoOne(t *testing.T) {
 
 	expectedSpanIds := make(map[int][]pdata.SpanID)
 	expectedSpanIds[0] = []pdata.SpanID{
-		tracetranslator.UInt64ToSpanID(uint64(1)),
+		// TODO
+		// tracetranslator.UInt64ToSpanID(uint64(1)),
```

Comment:
Not feeling very comfortable with keeping this commented, there should be a simple way to translate those even if the function got removed

Reply:
Done, copied over code from idutils related to big endian conversion